BCStudentSoftwareDevTeam/celts
# Changed language on manage volunteers button and modal to be more descriptive

## Issue
Fixes issue #1123 

The language on the button and the modal of the `Manage Volunteers` page to add users to an event was poor. We went through and made it consistent and make more sense from a first-timer's perspective.

## Changes
- Made the text that the `Add Volunteers` button on the `Manage Volunteers` page respond to whether the event is past, the rsvp list is full, or rsvp is required. 
- Changed the wording on all the buttons and modal titles to ensure that our language was consistent.
- Did a teeny refactor to improve code reusability.

## Testing
- switch to `1123_fix_volunteer_modal_language`\
- `source setup.sh` and `./database/reset_database.sh test`
- Create a past event and navigate to the manage volunteers page.
- Make sure that the button says `Add Volunteers`, and that the modal has the same language.
- Make the event a future event and set the rsvp limit to 1.
- Navigate back to the manage volunteers page.
- Make sure that the button says `RSVP Volunteers`, and that the modal has the same language.
- Add a user like Sreynit to the RSVP list, now the limit should be hit and the waitlist should be active.
- Make sure that the button says `Waitlist Volunteers`, and that the modal has the same language.
- Finally, check that if you add multiple volunteers at the same time through the modal the Add/RSVP/Waitlist button turns plural.
